src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -1149,7 +1160,7 @@ CREATE TABLE transient_object_directory
 CREATE TABLE user_account
 (usr_id                      INTEGER                       NOT NULL
 ,dm_user_s_object_id         CHARACTER VARYING(16)
-,user_name                   CHARACTER VARYING
+,user_name                   CHARACTER VARYING             NOT NULL
 ,user_email_address          CHARACTER VARYING

```

Comment:
@barry-jones50410 I thought `user_email_address` would also be NOT NULL?
Please can you clarify what is expected in the `user_name` column as I see the Dev Team using email address currently in both columns on staging database - [Populate User / Role / Permissions](https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1712514251)

As mentioned in the Darts Data Model channel on 08/09:
> I know we're using the `user_email_address` currently for the external users to populate the SecurityToken.userState as part of `/external-user/handle-oauth-code`.
> I think `user_name` might be something like a display name e.g. "Richard B"?